csrc/python_frontend/python_bindings.cpp

```diff
@@ -2726,6 +2735,48 @@ void initNvFuserPythonBindings(PyObject* module) {
   py::class_<FusionDefinition::SchedOperators> nvf_sched(
       fusion_def, "SchedOperators");
   nvf_sched.def(py::init<FusionDefinition*>());
+  //! experimental API for multidevice support
+  nvf_sched.def(
+      "_create_device_mesh",
+      [](FusionDefinition::SchedOperators& self, std::vector<int64_t> devices) {
+        FUSER_PERF_SCOPE("SchedOperators._create_device_mesh");
```

Comment:
Why FUSER_PERF_SCOPE? Is the constructor of DeviceMesh expensive enough to justify the overhead of scoping?